website/.gitignore

```diff
@@ -3,3 +3,6 @@ node_modules
 .next
 out
 .mdx-data
+
+# Expose non-sensitive environment variables here
+!.env
```

Comment:
I tend to agree with you on this. As a team we've adopted a slightly different approach where we check in our `.env` to version control to avoid the `.env.sample` runaround when the keys it holds are fine for public consumption.

The question here remains whether or not these keys should even exist in a `.env` file to being with. I'm willing to think through this a bit.

What I haven't shown is the UI portion of this code which does expect environment variables to be public, but we can expose them through different means, namely the `next.config.js` file

cc/ @mwickett

Reply:
Leveraging the conventions advised by Next.js documented here: https://nextjs.org/docs/basic-features/environment-variables